pkg/piecestore/rpc/client/pieceranger.go

```diff
@@ -67,10 +67,22 @@ func (r *pieceRanger) Range(ctx context.Context, offset, length int64) (io.ReadC
 		return ioutil.NopCloser(bytes.NewReader([]byte{})), nil
 	}
 
+	stream := r.stream
```

Comment:
What's the point of this helper variable. Can't you do all the below logic referring directly to `r.stream`?